index.js

```diff
@@ -84,10 +84,18 @@ module.exports = (input, opts) => {
 		});
 	}
 
-	tasks.add({
-		title: 'Bumping version',
-		task: () => exec('npm', ['version', input])
-	});
+	tasks.add([
+		{
+			title: 'Bumping version using Yarn',
```

Comment:
This makes me wonder, why don't we just go for 

```js
tasks.add([
    {
        title: `Bumping version using ${opts.yarn === true ? 'Yarn' : 'npm'}`,
        task: () => exec(opts.yarn === true ? 'yarn' : 'npm', ['version', input])
    }
]);
```

Might have a reason though but can't recall.